artist_query/lib/artist_query.ex

```diff
@@ -0,0 +1,41 @@
+defmodule ArtistQuery do
+  def main(_args) do
+    artist = prompt
+    load_songs("music.csv")
+    |> Enum.filter_map(&(String.upcase(&1["Artist"])) == artist, &(&1["Name"]))
+    |> put_songs(titlecase(artist))
+
+    main([]) # Repeat forever
```

Comment:
Looks like you aren't using this argument for the body of `main`, but you still have to pass in this empty list. Perhaps this is indicative that you should pull out a new function with 0 arity (`main/1` would just call that function)